app/jobs/application_job.rb

```diff
@@ -11,17 +11,24 @@
 # @see [resque-lock] https://github.com/defunkt/resque-lock/blob/master/lib/resque/plugins/lock.rb
 # @see [redis setnx] http://redis.io/commands/setnx
 class ApplicationJob < ActiveJob::Base
-  extend Resque::Plugins::Lock
+  include Resque::Plugins::Lock
```

Comment:
why did you change to `include`? The documentation says to `extend` https://github.com/defunkt/resque-lock#resque-lock

Reply:
Based on my (perhaps limited understanding), extends added the functions as class methods. As such, they ignored an override of the lock function (L29), which was also suggested by https://github.com/defunkt/resque-lock#resque-lock. It seems that include is the proper way to mixin the Lock module (unless it's not).